kroxylicious-systemtests/src/main/java/io/kroxylicious/systemtests/Constants.java

```diff
@@ -140,6 +140,8 @@ public interface Constants {
      * Test clients image url
      */
     String TEST_CLIENTS_IMAGE = "quay.io/strimzi-test-clients/test-clients:latest-kafka-" + Environment.KAFKA_VERSION;
+    String KCAT_CLIENT_IMAGE = "quay.io/kroxylicious/kcat:1.7.1";
+    String KAF_CLIENT_IMAGE = "quay.io/kroxylicious/kaf";
```

Comment:
we should specify a version so that the tests are stable over time.

Reply:
+1